osmnx/utils_graph.py

```diff
@@ -304,7 +304,7 @@ def remove_isolated_nodes(G):
     G = G.copy()
 
     # get the set of all isolated nodes, then remove them
-    isolated_nodes = {node for node, degree in G.degree() if degree < 1}
+    isolated_nodes = {node for node, degree in G.degree() if not degree}
```

Comment:
Please undo this change. When handling integers, implicit false may involve more risk than benefit and makes the code less readable.

Reply:
Thank you, done.